R/PictureBlock.R

```diff
@@ -20,21 +20,28 @@ PictureBlock <- R6::R6Class( # nolint: object_name_linter.
     #'
     #' @details throws if argument is not a `ggplot`, `grob` or `trellis` plot.
     #'
-    #' @param content (`ggplot`, `grob`, `trellis`) a picture in this `PictureBlock`
+    #' @param content (`ggplot` or `grob` or `trellis`) a picture in this `PictureBlock`
+    #'
+    #' @return `self`, invisibly.
+    #' @examplesIf requireNamespace("ggplot2") && requireNamespace("lattice")
+    #' ## ------------------------------------------------
```

Comment:
There's a caveat with using `@examplesIf` with methods as "technically" it's not a supported tag (see https://github.com/r-lib/roxygen2/issues/1521)

The problem? The example for the specific method does not appear as images show.

Should we continue to use this as it is or:
1. Revert to `@examples` (as it was before)
2. Revert to `@examples` and change to a big if clause that checks for `requireNamespace`

![image](https://github.com/insightsengineering/teal.reporter/assets/211358/f712bfe0-06b9-48ed-8e4a-a02305adf1e3)

![image](https://github.com/insightsengineering/teal.reporter/assets/211358/479ce916-5737-4e76-9e12-781dbf7b31b3)

Reply:
I reverted this to `@examples`